components/handlers/module_edit_preclusions.py

```diff
@@ -0,0 +1,46 @@
+'''
+    This module contains the handler for web requests pertaining to
+    the editing of a module's preclusions.
+'''
+
+
+import json
+from app import RENDER
+import web
+from components import model, session
+
+
+class EditModulePreclusions(object):
+    '''
+        This class handles the editing of a module's preclusions.
+    '''
+    def GET(self):
+        '''
+            Handles the loading of the 'Edit Module Preclusions' page.
+        '''
+        if not session.validate_session():
+            raise web.seeother('/login')
+        else:
+            input_data = model.validate_input(web.input(), ["code"])
+            module_code = input_data.code
+
+            module_info = model.get_module(module_code)
```

Comment:
Line 27-30 are not necessary because model.validate_input() is already checking whether the module exists. Same applies to the pre-req handler